src/ResourceManager/AnalysisServices/AzureRM.AnalysisServices.ServiceManagement.psd1

```diff
@@ -0,0 +1,108 @@
+﻿#  
```

Comment:
This is a different manifest than the AzureRM.AnalysisServices.psd1. So we need some differentiation...I renamed AzureRM.AnalysisServices.ServiceManagement.psd1 to AzureRM.AnalysisServices.Dataplane .psd1 which is shorter